src/components/breadcrumbs/_breadcrumbs.scss

```diff
@@ -25,11 +25,14 @@
 
 .ouiBreadcrumb {
   display: inline-block;
-  margin-bottom: $ouiSizeXS; /* 1 */
+  transform: skewX(20deg);
 
   &:not(.ouiBreadcrumb--last) {
-    margin-right: $ouiBreadcrumbSpacing;
     color: $ouiTextSubduedColor;
+
+    &:hover {
+      color: $ouiBreadCrumbHoverColor !important; // sass-lint:disable-line no-important
```

Comment:
Why all the `important` declarations? We should reconsider the class names and other styles that are causing these conflicts rather than just sledgehammering them.

Reply:
+1 important should be avoided.

Reply:
Created an issue for this: https://github.com/opensearch-project/oui/issues/376